audit/conn/internal/writer/writer_test.go

```diff
@@ -0,0 +1,35 @@
+package writer
+
+import (
+	"bytes"
+	"context"
+	"net"
+	"testing"
+
+	"github.com/microsoft/go-otel-audit/audit/msgs"
+)
+
+type fakeConn struct {
+	net.Conn
+
+	buff *bytes.Buffer
+}
+
+func (f *fakeConn) Write(b []byte) (int, error) {
+	f.buff.Write(b)
```

Comment:
minor: Should we be propagating error here?

Reply:
I guess the only way this fails is OOM?

Reply:
Pretty much, so I was skipping it.  But I went ahead and fixed this.